common/dynamicconfig/constants.go

```diff
@@ -91,6 +91,8 @@ const (
 	EnableActivityEagerExecution = "system.enableActivityEagerExecution"
 	// NamespaceCacheRefreshInterval is the key for namespace cache refresh interval dynamic config
 	NamespaceCacheRefreshInterval = "system.namespaceCacheRefreshInterval"
+	// EnableBatcherPerNamespaceWorker decides whether start batcher worker per namespace
+	EnableBatcherPerNamespaceWorker = "system.enableBatcherPerNamespaceWorker"
```

Comment:
maybe
```suggestion
	EnableBatcher = "system.enableBatcher"
```
?

Reply:
It's still possible to enable/disable each worker component (scheduler, batcher) per-namespace, even though you can't change the count. So we could have another dynamic config for that. I don't feel strongly either way.